presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -140,7 +139,56 @@ public void testMySqlCreatedParametrizedVarcharUnicode()
                 .addRoundTrip(varcharDataType(sampleUnicodeText.length(), CHARACTER_SET_UTF8), sampleUnicodeText)
                 .addRoundTrip(varcharDataType(32, CHARACTER_SET_UTF8), sampleUnicodeText)
                 .addRoundTrip(varcharDataType(20000, CHARACTER_SET_UTF8), sampleUnicodeText)
-                .execute(queryRunner, new CreateAndInsertDataSetup(mysql, "tpch.mysql_test_parameterized_varchar_unicode"));
+                .execute(queryRunner, mysqlCreateAndInsert("tpch.mysql_test_parameterized_varchar_unicode"));
+    }
+
+    @Test
+    public void testPrestoCreatedParametrizedChar()
+            throws Exception
+    {
+        mysqlCharTypeTest().execute(queryRunner, prestoCreateAsSelect("mysql_test_parameterized_char"));
+    }
+
+    @Test
+    public void testMySqlCreatedParametrizedChar()
+            throws Exception
+    {
+        mysqlCharTypeTest()
+                .addRoundTrip(charDataType("char", 1), "")
```

Comment:
because it fails when trying to create the char type, I think. I'll have a look and show you live :)